mappings/net/minecraft/world/ChunkRegion.mapping

```diff
@@ -9,14 +9,14 @@ CLASS net/minecraft/class_3233 net/minecraft/world/ChunkRegion
 	FIELD field_14094 fluidTickScheduler Lnet/minecraft/class_1951;
 	FIELD field_14096 dimension Lnet/minecraft/class_2874;
 	FIELD field_14097 levelProperties Lnet/minecraft/class_5217;
-	FIELD field_14098 chunks Ljava/util/List;
+	FIELD field_14098 surroundingChunks Ljava/util/List;
```

Comment:
`surroundingChunks` in this context feels like it implies only the edge chunks. I think `chunks` might be descriptive enough?

Reply:
maybe neighboring chunks?

Reply:
In the context of `ChunkStatus` that makes sense, but I think `ChunkRegion` is used more generically. It's just all the chunks contained within the area. If `chunks` doesn't work, maybe `containedChunks`/`includedChunks` or something along those lines?

Reply:
On further thought, `chunks` should be fine